openreview/venue/venue.py

```diff
@@ -1009,6 +1015,125 @@ def set_SAC_ethics_review_process(self, sac_ethics_flag_duedate=None):
     def open_reviewer_recommendation_stage(self, start_date=None, due_date=None, total_recommendations=7):
         self.invitation_builder.set_reviewer_recommendation_invitation(start_date, due_date, total_recommendations)
 
+    def ithenticate_create_and_upload_submission(self):
+        if not self.iThenticatePlagiarismCheck:
+            raise openreview.OpenReviewException('iThenticatePlagiarismCheck is not enabled for this venue.')
+        
+        self.invitation_builder.set_iThenticate_plagiarism_check_invitation()
+
+        iThenticate_client = openreview.api.iThenticateClient(self.iThenticatePlagiarismCheckApiKey, self.iThenticatePlagiarismCheckApiBaseUrl)
+        
+        submissions = self.get_submissions()
+        for submission in submissions:
+            # Create a new submission ID in iThenticate
+            # TODO - Decide what should go in metadata.group_context.owners
+            res = iThenticate_client.create_submission(
```

Comment:
I think before creating the submission, we need to check if there is already an edge associated with the openreview note id. Otherwise we can duplicate the requests, same head with multiple tails.

Reply:
In that case, would you suggest deleting the submission and edge? Or, should we skip creating a new edge and change the tail in the existing edge to the new submission ID (and delete the previous submission ID)?

Reply:
That's depends on the edge.label. 

If the label is error then we can submit a new submission but if there is another state then I would just skip that submission.

Reply:
Added the change